generators/client/templates/angular/tsconfig.json.ejs

```diff
@@ -27,6 +27,9 @@
     "strict": true,
     "noImplicitReturns": true,
     "noFallthroughCasesInSwitch": true,
+    <%_ if (protractorTests) { _%>
```

Comment:
Probably we should add a description for the reason.
```suggestion
    <%_ if (protractorTests) { // Prevents Jest and Mocha types conflicts. See https://github.com/jhipster/generator-jhipster/pull/13083 _%>
```

Reply:
It's good idea, I was afk and just 1 minute before clicking suggestion accept you merged this :) I'm doing new PR to add this comment here.